assets/components/package.json

```diff
@@ -1,51 +1,51 @@
 {
-  "name": "newspack-components",
-  "version": "1.6.1",
-  "description": "Newspack design system components",
-  "author": "Automattic",
-  "license": "GPL-2.0-or-later",
-  "keywords": [
-    "wordpress",
-    "newspack",
-    "components"
-  ],
-  "bugs": {
-    "url": "https://github.com/Automattic/newspack-plugin/issues"
-  },
-  "homepage": "https://github.com/Automattic/newspack-plugin",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/Automattic/newspack-plugin.git",
-    "directory": "assets/src/components"
-  },
-  "main": "dist/cjs/index.js",
-  "publishConfig": {
-    "access": "public"
-  },
-  "dependencies": {
-    "@wordpress/base-styles": "^3.5.1",
-    "@wordpress/components": "^12.0.6",
-    "@wordpress/element": "^2.19.0",
-    "@wordpress/i18n": "^3.17.0",
-    "@wordpress/icons": "^4.0.1",
-    "classnames": "^2.2.6",
-    "lodash": "^4.17.20",
-    "qs": "^6.9.6",
-    "react-router-dom": "^5.2.0"
-  },
-  "devDependencies": {
-    "@automattic/calypso-build": "^6.3.0",
-    "@babel/cli": "^7.0.0",
-    "@babel/core": "^7.12.10",
-    "recursive-copy": "2.0.10"
-  },
-  "babel": {
-    "ignore": [
-      "./node_modules"
-    ]
-  },
-  "scripts": {
-    "prepublishOnly": "transpile && node copy-styles.js",
-    "postpublish": "rm -r dist && rm -r shared"
-  }
+	"name": "newspack-components",
+	"version": "1.7.0",
+	"description": "Newspack design system components",
+	"author": "Automattic",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"newspack",
+		"components"
+	],
+	"bugs": {
+		"url": "https://github.com/Automattic/newspack-plugin/issues"
+	},
+	"homepage": "https://github.com/Automattic/newspack-plugin",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/newspack-plugin.git",
+		"directory": "assets/src/components"
+	},
+	"main": "dist/esm/index.js",
+	"publishConfig": {
+		"access": "public"
+	},
+	"dependencies": {
+		"@wordpress/base-styles": "^4.0.0",
+		"@wordpress/components": "^17.0.0",
```

Comment:
I think new versions have come out in the meantime (v18 is available), so it might be worth updating again to avoid having to do it later

Reply:
Like trying to hit a moving target! This PR implements Dependabot, so I might freeze the updates as they are right now since things are currently building and functional.

Reply:
cool, that also sounds like a good plan!

> Like trying to hit a moving target

Yeah, it can be very difficult 😬